gatsby-ssr.js

```diff
@@ -11,22 +11,22 @@ export const onRenderBody = ({ setHtmlAttributes, setHeadComponents }) => {
 
   setHtmlAttributes(HtmlAttributes);
 
-  if (process.env.NODE_ENV === `production`) {
```

Comment:
Errrr... that's definitely not right. We don't want to lead the plausible script in any other env than `production`